src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_NotificationsScreen.tsx

```diff
@@ -0,0 +1,138 @@
+import React, {useEffect} from 'react';
+import {FullScreenView} from '@atb/components/screen-view';
+import {Section, ToggleSectionItem} from '@atb/components/sections';
+import {Linking, View} from 'react-native';
+import {ThemeText} from '@atb/components/text';
+import {StaticColorByType} from '@atb/theme/colors';
+import {StyleSheet} from '@atb/theme';
+import {dictionary, ProfileTexts, useTranslation} from '@atb/translations';
+import {MessageBox} from '@atb/components/message-box';
+import {Processing} from '@atb/components/loading';
+import {useIsFocusedAndActive} from '@atb/utils/use-is-focused-and-active';
+import {isConfigEnabled} from '@atb/push-notifications/utils';
+import {usePushNotifications} from '@atb/push-notifications';
+
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+export const Profile_NotificationsScreen = () => {
+  const style = useStyles();
+  const {t} = useTranslation();
+  const isFocusedAndActive = useIsFocusedAndActive();
+  const {
+    permissionStatus,
+    isLoading,
+    isUpdating,
+    isError,
+    config,
+    register,
+    checkPermissions,
+    updateConfig,
+  } = usePushNotifications();
+
+  useEffect(() => {
+    if (isFocusedAndActive) {
+      checkPermissions();
+    }
+  }, [isFocusedAndActive, checkPermissions]);
+
+  const handlePushNotificationToggle = async (enabled: boolean) => {
+    if (enabled) {
+      await register();
```

Comment:
I remember Martin talking something about registering on every app startup? You have discussed with him when, and how often, the register action should take place?

Reply:
Yes, I remember there was some talking about that. `register()` can be called as many times as you want. It can be done at startup, in onboarding, or here. The backend doesn't care. The `use-push-notifications` hook is designed in a way that one can add `const {register} = usePushNotifications()` and call `register` at any time.

Adding a call to `register` at startup is beyond the scope of this pr.

Reply:
Handled in https://github.com/AtB-AS/kundevendt/issues/7230